src/editor/actions/actions.js

```diff
@@ -99,46 +65,34 @@ export function updateJsmdContent(text, autosaveChanges = true) {
       jsmd: text,
       jsmdChunks
     });
-
-    // queue an update to autosave if applicable
-    if (autosaveChanges) {
-      dispatch(updateAutosave());
-    }
   };
 }
 
-export function setPreviousAutosave(hasPreviousAutoSave) {
-  return {
-    type: "SET_PREVIOUS_AUTOSAVE",
-    hasPreviousAutoSave
-  };
-}
-
-export function loadAutosave() {
-  return async (dispatch, getState) => {
-    // jsmd, jsmdChunks
-    const { dirtyCopy: jsmd } = await getLocalAutosaveState(getState());
-    const jsmdChunks = jsmdParser(jsmd);
-    dispatch({
-      type: "REPLACE_NOTEBOOK_CONTENT",
-      jsmd,
-      jsmdChunks
-    });
-    dispatch(setPreviousAutosave(false));
-  };
+export function toggleWrapInEditors() {
+  return { type: "TOGGLE_WRAP_IN_EDITORS" };
 }
 
-export function discardAutosave() {
-  return (dispatch, getState) => {
-    clearLocalAutosave(getState());
-    dispatch(setPreviousAutosave(false));
+export function makeNotebookCopy(forkedFrom) {
```

Comment:
not sure if this is a candidate for moving out of actions.js too? seems like a saving related action.

Reply:
Turns out this wasn't being used anymore, so we can just delete it! That was easy.